app.py

```diff
@@ -16,6 +16,17 @@
 import src.settings
 import src.swap_amm
 
+# color map
+COLORS_MAP = {
```

Comment:
You created this class and use only `Collateral` and `Debt` why do you need others key:value?